Ginger/Ginger/Run/GingerRunner.cs

```diff
@@ -2055,7 +2084,7 @@ private void DoFlowControl(Act act)
                                     {
                                         ActSetVariableValue setValueAct = new ActSetVariableValue();
                                         setValueAct.VariableName = vals[0];
-                                        setValueAct.SetVariableValueOption = ActSetVariableValue.eSetValueOptions.SetValue;
+                                        setValueAct.SetVariableValueOption = eSetValueOptions.SetValue;
```

Comment:
Since this enum is relevant only for ActSetVariableValue please put it back

Reply:
Done